packages/trader/src/Modules/Contract/Components/AccumulatorsStats/__tests__/accumulators-stats-manual-modal.spec.tsx

```diff
@@ -1,18 +1,29 @@
 import React from 'react';
-import { fireEvent, render, screen, within } from '@testing-library/react';
+import { render, screen, within } from '@testing-library/react';
 import { AccumulatorsStatsManualModal } from '../accumulators-stats-manual-modal';
+import userEvent from '@testing-library/user-event';
+
+type TModal = React.FC<{
```

Comment:
```suggestion
type TModal = React.FC<{
```
are we using React.FC?

Reply:
what are we using instead ?

Reply:
Hamza it okay to use React.FC i guess. [Thread discussion ](https://deriv-group.slack.com/archives/C043LHF58AC/p1684143763078919?thread_ts=1684125975.008349&cid=C043LHF58AC)